feature-extraction/ohdsi/feature_extraction/__init__.py

```diff
@@ -22,6 +24,249 @@ def _py_none_to_null(py_obj):
     return robjects.NULL
 
 
+class Aggregation:
+
+    @staticmethod
+    def aggregate_covariates (covariate_data) -> RS4:
```

Comment:
* Remove space after function name
* Add type hint for `covariate_data` (`RS4`?)